Newsletter-266: translate into Chinese (#74)
PrimitivesLane/bitcoinops.github.io: _posts/zh/newsletters/2023-08-30-newsletter.md

---
title: 'Bitcoin Optech Newsletter #266'
permalink: /zh/newsletters/2023/08/30/
name: 2023-08-30-newsletter-zh
slug: 2023-08-30-newsletter-zh
type: newsletter
layout: newsletter
lang: zh
---
本周的周报包括了一项老的闪电网络实现中漏洞的尽责披露的公告，总结了对提议的限制条款的操作码进行混合的建议；此外还包括我们的常规内容内容以及 Bitcoin Stack Exchange 中的精选问答、新软件发布和候选发布的公告，以及热门比特币基础设施项目的重大变更的汇总。

## 新闻

> - **<!--disclosure-of-past-ln-vulnerability-related-to-fake-funding-->此前与假充值相关的闪电网络漏洞的信息披露：** Matt Morehouse [发布][morehouse dos]到 Lighting-Dev 邮件列表中，总结了他之前[尽责披露][topic responsible disclosures]的漏洞。该漏洞现在在所有流行的闪电网络实现的最新版本中都已经解决。为理解该漏洞，我们假设 Bob 运行了一个闪电网络节点。他收到 Mallory 节点的请求，要求打开一个新的通道。他们会经历通道开启的过程，直到 Mallory 该广播通道充值交易的时候。为了以后使用该通道，Bob 需要存储与之相关的一些状态，并开始扫描能让交易得到足够多确认的新区块。如果 Mallory 再也不广播该交易，就会浪费掉 Bob 的存储和扫描资源。而如果 Mallory 重复这个过程数千次或数百万次，可能会浪费 Bob 的资源，以至于他的闪电网络节点无法执行任何其他操作，包括需要及时执行的防止资金损失的操作。

Reviewer: ```suggestion
- **<!--disclosure-of-past-ln-vulnerability-related-to-fake-funding-->此前与假充值相关的闪电网络漏洞的信息披露：** Matt Morehouse [发布][morehouse dos]到 Lighting-Dev 邮件列表中，总结了他之前[尽责披露][topic responsible disclosures]的漏洞。该漏洞现在在所有流行的闪电网络实现的最新版本中都已经解决。为理解该漏洞，我们假设 Bob 运行了一个闪电网络节点。他收到 Mallory 节点的请求，要求打开一个新的通道。他们会经历通道开启的过程，但停在需要 Mallory 广播通道充值交易的阶段。为了以后使用该通道，Bob 需要存储与之相关的一些状态，并开始扫描能让交易得到足够多确认的新区块。如果 Mallory 再也不广播该交易，就会浪费掉 Bob 的存储和扫描资源。而如果 Mallory 重复这个过程数千次或数百万次，可能会浪费 Bob 的资源，以至于他的闪电网络节点无法执行任何其他操作，包括需要及时执行的防止资金损失的操作。
```

- [Bitcoin Core #27460][] 添加了一个新的 `importmempool` RPC。该 RPC 将加载一个 `mempool.dat` 文件，并尝试将其中加载的交易添加到节点的交易池中。{% assign timestamp="51:16" %}

- [LDK #2248][] 提供了一个内置系统，LDK 的下游项目可以用其来跟踪在 gossip 消息中引用的 UTXO。处理 gossip 消息的闪电网络节点只能接受和 UTXO 相关联的密钥所签名的消息，否则它们可能会被迫处理和转发垃圾消息，或者尝试通过不存在的通道转发支付（尝试将始终失败）。新内置的 `UtxoSource` 可用于连接到本地 Bitcoin Core 的闪电网络节点。{% assign timestamp="53:43" %}

- [LDK #2337][] 使得更加容易地使用 LDK 来构建[瞭望塔][topic watchtowers]。瞭望塔是独立于用户钱包运行，但可以接收来自用户节点的加密闪电网络惩罚交易。瞭望塔可以从新区块中的每笔交易中提取信息，并使用该信息尝试解密先前接收到的加密数据。如果解密成功，瞭望塔可以广播解密后的惩罚交易。当用户不在线时，可以保护用户免受对方发布已撤销的通道状态的影响。{% assign timestamp="55:00" %}

- [LDK #2411][] 和 [#2412][ldk #2412] 添加了一个为[盲化支付][topic rv routing]构建支付路径的 API。这些 PRs 帮助将 LDK 的[洋葱消息][topic onion messages]的代码（其中使用了盲化路径）与盲化路径本身分离开。[#2413][ldk #2413] 中的一项跟进实际上将添加对盲化路径的支持。{% assign timestamp="1:00:16" %}

- [LDK #2507][] 添加了对另一个实现中长期存在并会导致通道非必要的强制关闭问题的变通解决方法。{% assign timestamp="1:04:46" %}

> - [LDK #2478][] 添加了一个此前转发过的 [HTLC][topic htlc] 在当前被结算时的事件。该事件可提供这个 HTLC 的相关信息，包括它来自哪个通道、HTLC 的金额以及从中收取的手续费金额。{% assign timestamp="1:07:22" %}

Reviewer: ```suggestion
- [LDK #2478][] 添加了一个事件，用于在结算此前转发过的 [HTLC][topic htlc] 时提供信息。该事件可提供这个 HTLC 的相关信息，包括它来自哪个通道、HTLC 的金额以及从中收取的手续费金额。{% assign timestamp="1:07:22" %}
```

> - [LND #7904][] 添加了对“简单 taproot 通道”的实验性支持，允许闪电网络的充值和承诺交易使用 [P2TR][topic taproot]，并支持当双方合作时使用无脚本[多签][topic multisignature]签署的 [MuSig2][topic musig]。这减少了交易的权重空间，并在通道合作关闭时提高了隐私。LND 继续专门使用 [HTLCs][topic htlc]，允许从 taproot 通道开始的支付继续通过其他不支持 taproot 通道的闪电网络节点进行转发。{% assign timestamp="1:09:07" %}

Reviewer: ```suggestion
- [LND #7904][] 添加了对“简单 taproot 通道”的实验性支持，允许闪电网络的充值和承诺交易使用 [P2TR][topic taproot]，并支持当双方合作时使用无脚本[多签][topic multisignature]签署的 [MuSig2][topic musig]。这减少了交易的重量，并在通道合作关闭时提高了隐私。LND 继续只使用 [HTLCs][topic htlc]，允许从 taproot 通道开始的支付继续通过其他不支持 taproot 通道的闪电网络节点进行转发。{% assign timestamp="1:09:07" %}
```
